src/runner.js

```diff
@@ -137,6 +136,27 @@ define(function (require, exports) {
                 if (consoleOutput)
                     console.log(pp(call) + " -> " + pp(fn));
             });
+            if(this.args.output !== null){
+                let filename = this.args.output[0];
+                if(!filename.endsWith(".json")){
+                    filename += ".json";
+                }
+                fs.writeFile(filename, JSON.stringify(result, null, 2), function (err) {
+                    if (err) {
+                        /*
+                        When happened something wrong (usually out of memory when we want print
+                        the result into a file), then we try to file with JSONStream.
```

Comment:
Why don't we use JSONStream all the time? I don't think that would be a big overhead compared to the simple filewriting. Also, I prefer synchronized file writing.

Reply:
You right, but I think, that the streamed JSON is not possible write to file as pretty json like. We should find a solution for be possible it.
I think this more readable:
`
[
  {
    source: {
      label: '...',
      range: [0, 11]
    },
    target: {
      label: '...',
      range: [0, 11]
    }
  }
]
`

against this:
`
[  {    source: {      label: '...',      range: [0, 11]    },    target: {      label: '...',      range: [0, 11]    }  }]
`

If the operation require too big memory (and we are out of memory), only then use the JSONStream, but okay to me if others prefer this version :)

Reply:
Hi guys, I'll merge this version for now, and if out of memory turns out to be a big problem, then we can switch to always using JSONStream.